tests/helper_functions.py

```diff
@@ -211,6 +212,19 @@ def pyfunc_serve_and_score_model(
     activity_polling_timeout_seconds=500,
     extra_args=None,
     stdout=sys.stdout,
+):
+    with pyfunc_scoring_endpoint(
+        model_uri,
+        extra_args=extra_args,
+        activity_polling_timeout_seconds=activity_polling_timeout_seconds,
+        stdout=stdout,
+    ) as endpoint:
+        return endpoint.invoke(data, content_type)
+
+
+@contextmanager
+def pyfunc_scoring_endpoint(
```

Comment:
note: Splitting the server start and inference so we can reuse the same scoring server for multiple requests.

Reply:
Is it possible to land this change first to reduce diff and for easier review?

Reply:
Sure!

Reply:
Filed: https://github.com/mlflow/mlflow/pull/12581

Reply:
Thanks!